requirements.txt

```diff
@@ -46,3 +47,4 @@ wandb>=0.16.4
 # it is for CLI usage ONLY
 yamllint>=1.35.1
 referencing>=0.35.0
+instructlab-training>=0.0.4
```

Comment:
Could you please order the dependencies in alphanumeric order? It makes it easier to find dependencies.

Reply:
Sure.

Reply:
Fixed.